extensions/amp-access/0.1/amp-access-iframe.js

```diff
@@ -108,11 +114,51 @@ export class AccessIframeAdapter {
 
   /** @override */
   pingback() {
-    return this.connectedPromise_.then(() => {
+    return this.connect().then(() => {
       return this.messenger_.sendCommandRsvp('pingback', {});
     });
   }
 
+  /**
+   * @return {!Promise}
+   * @package Visible for testing only.
+   */
+  connect() {
```

Comment:
tests for `connect` and `resolveConfig_` ?

Reply:
The test section for both is in the `test-amp-access-iframe.js` within the `describe('runtime connect', () => {...}`. PTAL.